### PR TITLE
[ADD] Add check double bookable partners

### PR DESCRIPTION
**Pull request by rimbendhaou**

### CODE REVIEW

File: project_resource_calendar/models/calendar_event.py [hunk 1]
```diff
@@ -161,7 +161,7 @@ def _get_weekday_number(self):
             ).weekday()
 
     @api.multi
-    @api.constrains('room_id', 'start', 'stop', 'equipment_ids')
+    @api.constrains('room_id', 'start', 'stop', 'equipment_ids', 'partner_ids')
```
Comment (rimbendhaou) [inline project_resource_calendar/models/calendar_event.py]: partner_ids?

**Review by rimbendhaou — COMMENTED** [review]
work in progress wainting for client feedback

**Comment by rimbendhaou** [comment]
waiting for client feedback

**Comment by rimbendhaou** [comment]
waiting for client approve

**Comment by rimbendhaou** [comment]
@wbeverly waiting for client feedback

**Comment by rimbendhaou** [comment]
@wbeverly Check feedback from client

**Comment by rimbendhaou** [comment]
@wbeverly check client specification

**Comment by rimbendhaou** [comment]
@wbeverly waiting for client feedback

File: project_resource_calendar/models/calendar_event.py [hunk 2]
```diff
@@ -172,7 +172,10 @@ def _check_resources_double_book(self):
             equipment = record.equipment_ids.filtered(
                 lambda s: s.allow_double_book is False
             )
-            if not any(room) and not any(equipment):
+            attendees = record.partner_ids.filtered(
+                lambda s: s.allow_double_book is False
+            )
+            if not any(room) and not any(equipment) and not any(attendees):
```
Comment (eilst) [inline project_resource_calendar/models/calendar_event.py]:
```suggestion
            if not room and not equipment and not attendees:
```

**Review by eilst — APPROVED** [review]

**Pull request merged**